builder/main.py

```
@@ -148,24 +193,34 @@ def _parse_partitions(env):
     return result
 
+
 def _update_max_upload_size(env):
+    """
+    Update the maximum upload size based on partition table configuration.
+    Prioritizes user-specified partition names.
+    """
     if not env.get("PARTITIONS_TABLE_CSV"):
         return
+
     sizes = {
```

Flake8 (7.2.0)

[error] local variable 'sizes' is assigned to but never used

(F841)